Libraries/Text/TextInput/RCTBackedTextInputDelegateAdapter.m

```diff
@@ -352,9 +357,11 @@ - (BOOL)textView:(NSTextView *)textView doCommandBySelector:(SEL)commandSelector
 {
   BOOL commandHandled = NO;
   id<RCTBackedTextInputDelegate> textInputDelegate = [_backedTextInputView textInputDelegate];
-  // enter/return
-  if (textInputDelegate.textInputShouldReturn && (commandSelector == @selector(insertNewline:) || commandSelector == @selector(insertNewlineIgnoringFieldEditor:))) {
-    [_backedTextInputView.window makeFirstResponder:nil];
+  // cmd + enter/return
+  if (commandSelector == @selector(noop:) && RCTEventIsCommandEnterEvent(NSApp.currentEvent)) {
```

Comment:
Interesting: the system sends a `noop:` selector on Cmd+Enter?

Reply:
Yup, that was news to me as well :smile: